gc/base/GCExtensionsBase.hpp

```diff
@@ -706,6 +706,13 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 
 	uintptr_t darkMatterSampleRate;/**< the weight of darkMatterSample for standard gc, default:32, if the weight = 0, disable darkMatterSampling */
 
+#if defined(OMR_GC_IDLE_HEAP_MANAGER)
+	uintptr_t idleMinimumFree;   /** < percentage of free heap to be retained as committed, default=0 for gencon, complete tenture free memory will be decommitted */
+	uintptr_t lastGCFreeBytes;  /** < records the free memory size from last Global GC cycle */
+	uint32_t gcOnIdleRatio; /**< the percentage of allocation since the last GC allocation determines the invocation of global GC, default global GC is invoked if allocation is > 20% */
```

Comment:
value range of the variable "gcOnIdleRatio" is 0 - 100. It doesn't need 8 bytes (uintptr_t).  
The next variable "compactOnIdle) is of type bool. 
So, we define the variable gcOnIdleRatio as 4 bytes, both the variables gcOnIdleRation + compactOnIdle can be accomdated in same 64 bit word.